tray/src/com/dvt/main/creational/abstract_factory/factory/Model1Factory.java

```diff
@@ -0,0 +1,39 @@
+<<<<<<< HEAD
+package com.dvt.creational.abstract_factory.factory;
+
+import com.dvt.creational.abstract_factory.Part;
+import com.dvt.creational.abstract_factory.doors.Model1Doors;
+import com.dvt.creational.abstract_factory.hood.Model1Hood;
+import com.dvt.creational.abstract_factory.wheels.Model1Wheels;
+=======
+package creational.abstract_factory.factory;
+
+import creational.abstract_factory.Part;
+import creational.abstract_factory.doors.Model1Doors;
+import creational.abstract_factory.hood.Model1Hood;
+import creational.abstract_factory.wheels.Model1Wheels;
+>>>>>>> main
```

Comment:
Please fix merge conflict.
```suggestion
<<<<<<< HEAD
package com.dvt.creational.abstract_factory.factory;

import com.dvt.creational.abstract_factory.Part;
import com.dvt.creational.abstract_factory.doors.Model1Doors;
import com.dvt.creational.abstract_factory.hood.Model1Hood;
import com.dvt.creational.abstract_factory.wheels.Model1Wheels;
=======
package creational.abstract_factory.factory;

import creational.abstract_factory.Part;
import creational.abstract_factory.doors.Model1Doors;
import creational.abstract_factory.hood.Model1Hood;
import creational.abstract_factory.wheels.Model1Wheels;
>>>>>>> main
```